src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="container"
+      data-qa="big"
+    >
+      <div class="big-search">
+        <input
+          class="input input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <div
+        class="small-search"
+        data-qa="big"
```

Comment:
Both `data-qa` attributes in the `big-search` and `small-search` divs have the same value `big`. If these elements are meant to be distinguished by their `data-qa` attributes, consider using unique values for each.